Makefile

```diff
@@ -10,7 +10,7 @@ VERSION := $(shell git describe --tags --dirty --always)
 # To re-generate a bundle for other specific channels without changing the standard setup, you can:
 # - use the CHANNELS as arg of the bundle target (e.g make bundle CHANNELS=preview,fast,stable)
 # - use environment variables to overwrite this value (e.g export CHANNELS="preview,fast,stable")
-CHANNELS="4.9"
+CHANNELS="stable"
```

Comment:
Why was 4.9 used here before contrary to the comment above?  Or turning it on its head, why don't we use 4.10 now?  Similarly below for `DEFAULT_CHANNEL`.